metadata/proto/metadata.proto

```diff
@@ -403,13 +403,23 @@ message DFTransformation {
 message PrimaryData {
     oneof location {
         PrimarySQLTable table = 1;
+        PrimaryPath path = 2;
     }
 }
 
 message PrimarySQLTable {
     string name = 1;
 }
 
+message PrimaryPath {
+    // The path, may be an absolute or a relative path
+	string path = 1;
+    // The types of a single file, or all the files in a directory
+	string file_type = 2;
+    // Is this a directory or files or a single one
+	bool is_dir = 3;
```

Comment:
weird formatting, too many spaces or space/tab mix?

Reply:
I installed a linter for protobuf and checked this and didn't find any issues. I think this might just be the way it's being displayed  in GH.

Reply:
Can you verify the spacing is exactly the same and both using spaces or tabs?